apps/fluent-tester/src/RNTester/FabricTester.tsx

```diff
@@ -1,11 +1,11 @@
+import { Separator, StealthButton, Text } from '@fluentui/react-native';
+import { useTheme } from '@uifabricshared/theming-react-native';
```

Comment:
I'm curious why the reordering of the import statements? It isn't necessarily wrong, I'm just curious whether there were build issues or whether it was reordered to follow an alphabetical approach.

Reply:
VSCode reorders them alphabetically every time I save a file. Would you prefer I turn off this setting so it limits the number of changes made to the files?